Gaijinizer patch/Map408.txt

```diff
@@ -4,551 +4,687 @@
+When the warped space time returned to its normal form,
+they were no-where to be seen...#Ron
 > END STRING
 
 > BEGIN STRING
 …………ただの時間稼ぎさ
 
 この程度で諦めてくれるようなら
 そもそも、あいつらはここに姿を現さない。
-> CONTEXT: Map408/events/1/pages/0/314/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/314/Dialogue/enemy27
+...That only bought us some time.
 
+If that were enough to make them give up, they
+wouldn't have appeared here in the first place.#Ron
 > END STRING
 
 > BEGIN STRING
 今のボクにできるのはこれくらい……
 
 いまのうちに、準備を始めなくちゃ……
-> CONTEXT: Map408/events/1/pages/0/319/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/319/Dialogue/enemy27
+That's as much as I can do right now.
 
+I need to prepare myself while I have the chance.#Ron
 > END STRING
 
 > BEGIN STRING
 ち、チェシャ………
-> CONTEXT: Map408/events/1/pages/0/325/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/325/Dialogue/kazuya
+C-Cheshire...#Ron
 > END STRING
 
 > BEGIN STRING
 ……………うれしいな
 
 まだ、その名前で呼んでくれるなんて。
-> CONTEXT: Map408/events/1/pages/0/331/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/331/Dialogue/enemy29
+...Having you still call me by that name makes
+me happy.#Ron
 > END STRING
 
 > BEGIN STRING
 ごめんね、驚かせちゃって………
 
 急展開すぎて、頭の理解が追いつかないよね………
-> CONTEXT: Map408/events/1/pages/0/335/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/335/Dialogue/enemy29
+Sorry for spooking you like that...
 
+A lot's happened just now so I'll bet you can't
+process it all.#Ron
 > END STRING
 
 > BEGIN STRING
 チェシャ……
 
 君はいったい、何者なんだ………？
 それに、さっきの姿も………
-> CONTEXT: Map408/events/1/pages/0/339/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/339/Dialogue/kazuya
+Cheshire...
 
+Just who the hell are you...?
+And what was that form you had before...?#Ron
 > END STRING
 
 > BEGIN STRING
 ………………ボクの存在を正確に言語化しようとすると
 とても難しいし、かなり長くなる……
 
 できたとしても、上手く理解されるかどうか……
-> CONTEXT: Map408/events/1/pages/0/345/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/345/Dialogue/enemy29
+It'd be rather tricky and time-consuming to put
+what I am into words. And even if I could, I'm
+not sure whether you'd understand anyway.#Ron
 > END STRING
 
 > BEGIN STRING
 でも幸いなことに、
 キミの知識のなかで、ボクを超簡単に表す単語がある
 
 ちょっと大雑把すぎるかもしれないけど…
-> CONTEXT: Map408/events/1/pages/0/350/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map408/events/1/pages/0/350/Dialogue/enemy29
+But thankfully, you have the knowledge inside you
+to express what I am in a simple way. Though it
+does oversimplify things a little...#Ron
 > END STRING
 
 > BEGIN STRING
 ボクは、\\C[2]キミと介入者をつなぐ\\C[10]『装置』\\C[0]……
 
 \\C[10]『介入』のロジックを担う生命体だよ。
-> CONTEXT: Map408/events/1/pages/0/356/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map408/events/1/pages/0/356/Dialogue/enemy29
+\\C[2]I am the \\C[10]device \\C[2]that's connected to both you and the
+intervener...
 
+\\C[10]A life-form that bears the logic behind "interventions".#RonIffyWording
 > END STRING
```

Comment:
Note... The text is very explicit on the second part, I'll need to think about what I do here.